src/python/bot/untrusted_runner/host.py

```diff
@@ -157,10 +156,7 @@ def wrapped(*args, **kwargs):
         # For timeouts, which aren't fatal errors, resurface the right
         # exception.
         if 'TimeoutError' in repr(e):
-          # TODO(ochang): Replace with generic TimeoutError in Python 3.
-          # Use __str__ to avoid newstrs. TODO(ochang): Use plain str() once
-          # migrated to Python 3.
-          raise engine.TimeoutError(e.__str__())
+          raise TimeoutError(e)
```

Comment:
I think this should be TimeoutError(str(e)) to avoid any unexpected issues with implicit string conversion.

Reply:
Thanks, updated in the PR.